examples/print-everything/src/main.rs

```diff
@@ -0,0 +1,66 @@
+// example
+
+use arctic::H10MeasurementType;
+
+struct Handler;
+
+#[arctic::async_trait]
+impl arctic::EventHandler for Handler {
+    async fn battery_update(&self, battery_level: u8) {
+        println!("Battery: {}", battery_level);
+    }
+
+    async fn heart_rate_update(&self, _ctx: &arctic::PolarSensor, heartrate: arctic::HeartRate) {
+        println!("Heart rate: {:?}", heartrate);
+    }
+
+    async fn measurement_update(&self, _ctx: &arctic::PolarSensor, data: arctic::PmdRead) {
+        println!("data: {:?}", data.data());
```

Comment:
Since everything else is capitalized this should probably be too

Reply:
That makes sense.